darshan-util/pydarshan/darshan/experimental/plots/plot_access_histogram.py

```diff
@@ -33,44 +33,64 @@ def plot_access_histogram(report, mod, ax=None):
     else:
         fig = None
 
-    # TODO: change to report.summary
-    if 'mod_agg_iohist' in dir(report):
-        report.mod_agg_iohist(mod)
-    else:
-        print("Cannot create summary, mod_agg_iohist aggregator is not registered with the report class.")
 
     # defaults
     labels = ['0-100', '101-1K', '1K-10K', '10K-100K', '100K-1M', '1M-4M', '4M-10M', '10M-100M', '100M-1G', '1G+']
 
-    agg = report.summary['agg_iohist'][mod]
-    # TODO: can simplify the read/write vals below after
-    # support for python 3.6 is dropped
-    read_vals = [
-        agg['READ_0_100'],
-        agg['READ_100_1K'],
-        agg['READ_1K_10K'],
-        agg['READ_10K_100K'],
-        agg['READ_100K_1M'],
-        agg['READ_1M_4M'],
-        agg['READ_4M_10M'],
-        agg['READ_10M_100M'],
-        agg['READ_100M_1G'],
-        agg['READ_1G_PLUS']
-    ]
-
-    write_vals = [
-        agg['WRITE_0_100'],
-        agg['WRITE_100_1K'],
-        agg['WRITE_1K_10K'],
-        agg['WRITE_10K_100K'],
-        agg['WRITE_100K_1M'],
-        agg['WRITE_1M_4M'],
-        agg['WRITE_4M_10M'],
-        agg['WRITE_10M_100M'],
-        agg['WRITE_100M_1G'],
-        agg['WRITE_1G_PLUS']
-    ]
-
+    agg=record['counters']
```

Comment:
I would not call this variable `agg`, as it implies it is some sort of aggregated value. Maybe just `counters` instead?

This variable was originally called `agg` because these plotting routines were aggregating all records in a given report object, which makes sense. Now, we are just passing in a single "record" to this routine, which could correspond to a single file record from a Darshan log or to an accumulated set of records (i.e., a summary record). In any case, any sort of aggregation now takes place outside of the plotting routine so we should just view the input as a generic record whose values are plotted directly.

Reply:
I will change it!